file(configure doesn't respect destdir so you can't use it for installing things :^)
--
--- a/crates/c-api/cmake/install-headers.cmake
+++ b/crates/c-api/cmake/install-headers.cmake
@@ -11,7 +11,7 @@
 
 message(STATUS "Installing: ${dst}/wasmtime/conf.h")
 file(READ "${include_src}/wasmtime/conf.h.in" conf_h)
-file(CONFIGURE OUTPUT "${dst}/wasmtime/conf.h" CONTENT "${conf_h}"
+file(CONFIGURE OUTPUT "${include_src}/wasmtime/conf.h" CONTENT "${conf_h}"
      NEWLINE_STYLE CRLF)
 file(INSTALL "${include_src}/"
      DESTINATION "${dst}"
